host/frontend/host-orchestrator/operator.go

```diff
@@ -46,12 +51,26 @@ func main() {
 
 	setupDeviceEndpoint(pool, config, socketPath)
 	r := setupServerRoutes(pool, polledSet, config)
-
 	http.Handle("/", r)
-	log.Println("Client endpoint created")
-	if err := http.ListenAndServe(fmt.Sprint(":", port), nil); err != nil {
-		log.Fatal("ListenAndServe client: ", err)
-	}
+
+	wg := new(sync.WaitGroup)
+	wg.Add(2)
+
+	go func() {
+		log.Println("Client http endpoint created")
+		log.Fatal(http.ListenAndServe(fmt.Sprint(":", httpPort), nil))
+		wg.Done()
+	}()
+
+	go func() {
+		log.Println("Client https endpoint created.")
+		certPath := tlsCertDir + "/cert.pem"
+		keyPath := tlsCertDir + "/key.pem"
+		log.Fatal(http.ListenAndServeTLS(fmt.Sprint(":", httpsPort), certPath, keyPath, nil))
```

Comment:
Please leave a comment with a pointer to the docs that say it's safe to use http.ListenAndServe...() from different go routines.

Reply:
made explicit creation of server instances per each go-routine to avoid the concurrency concerns that using `http.ListenAndServe` from different go-routines raise.

Reply:
The server objects were not the problem, the problem is sharing the mux object, which this code still does. All you need to do is add a comment mentioning that object is thread safe. The code as written before was OK, and simpler and more readable than the new one.

Reply:
added a comment mentioning that DefaultServerMux is used by both servers and that http.ServeMux instances are thread-safe.